guardian/newsletters-nx
# Tiered Edit Access

## What does this change?

This change introduces 4 new edit access levels:

- TagEditor
- BrazeEditor
- OphanEditor
- SignUpPageEditor

The edit form presented to a user in based on their access level. `TagEditors` may edit tags and the tag workflow status, `BrazeEditors` may edit the Braze values and workflow status etc.

This allows us to limit access to the edit view for a newsletter based on user context and should facilitate our async update workflows which request an update from a team to update the status of some activity outside the application. 


## How to test

- Set the access level to test in the .env.local and run the application (this file is evaluated at start time so you will need to restart for each test)
- Verify that the edit form shows the limited view (based on your selected access level)


## How can we measure success?

That views and updates are limited based on user permission status

## Have we considered potential risks?

Should be OK - tested locally
